### PR TITLE
Add tooltipOptions to pass props to tippy

### PR DESCRIPTION
**Pull request by davidjb**
See #49; this is updated post-review

### CODE REVIEW

File: src/layout.js [hunk 1]
```diff
@@ -25,7 +25,13 @@ export function render({ callbacks, options, random, selection, words }) {
 		onWordMouseOver,
 		onWordMouseOut,
 	} = callbacks;
-	const { colors, enableTooltip, fontStyle, fontWeight } = options;
+	const {
+		colors,
+		enableTooltip,
+		tooltipOptions,
```
Comment (chrisrzhou) [inline src/layout.js]: after line 33.  Since everything is sorted by convention so far, let's keep at it :)

**Review by chrisrzhou — COMMENTED** [review]
Might need to pull/rebase changes on `main`

File: src/layout.js (hunk 1 above)
Comment (davidjb) [reply src/layout.js]: Cool, good catch.  Done and moved down.

**Comment by davidjb** [comment]
All done, but I noticed there's a mix of tabs and spaces in `options.mdx` - noticed it during the rebase as some lines above mine have spaces but the rest of the file is tabs.  I kept mine with tabs for the predominant status quo in that file but it seems like you're going from tabs to spaces for the whole codebase and would probably clean that file up eventually if that's what's happening.  Happy to switch if you'd like 😃

**Comment by chrisrzhou** [comment]
yes spaces over tabs! At some point I was using tabs, but tab-support didn't turn out great for file previews on various document renderers in webapps (e.g. Github previews).  Feel free to format those to spaces if it's a straightforward formatting!

File: docs/usage/options.mdx [hunk 2]
```diff
@@ -146,8 +146,8 @@ Use the Playground to edit and play around with some of these options!
 				"#9467bd",
 				"#8c564b",
 			],
-			enableTooltip: true,
 			deterministic: false,
+			enableTooltip: true,
```
Comment (chrisrzhou) [inline docs/usage/options.mdx:150]: 🙏 thanks! Humans make mistakes!

**Review by chrisrzhou — APPROVED** [review]
Feel free to lint/format anything you feel that makes the codebase consistent in another PR.  I appreciate you being mindful about making minimal code changes in this PR.

**Comment by davidjb** [comment]
I just created #57 to clean up the last tabs in the docs -- that's all the remaining tabs in the project.

**Pull request merged**